AutomatedTesting/Gem/PythonTests/automatedtesting_shared/base.py

```diff
@@ -128,7 +128,11 @@ def teardown():
             errors.append(TestRunError("FAILED TEST", error_str))
             if return_code and return_code != TestAutomationBase.TEST_FAIL_RETCODE: # Crashed
                 crash_info = "-- No crash log available --"
-                crash_log = os.path.join(workspace.paths.project_log(), 'error.log')
+                if sys.platform.startswith('linux'):
+                    crash_log = os.path.join(workspace.paths.project_log(), 'crash.log')
+                else:
+                    crash_log = os.path.join(workspace.paths.project_log(), 'error.log')
```

Comment:
don't add if/else branches for platforms. This creates unnecessary complexity especially when needing to support a new platforms. Prefer to just check if the initial log file exists, and if not, use the alternative one, like:
```suggestion
                crash_log = os.path.join(workspace.paths.project_log(), 'error.log')
                if not os.path.is_file(crash_log):
                     crash_log = os.path.join(workspace.paths.project_log(), 'crash.log')
```

Reply:
I disagree here. It's clearer to me why we're checking for crash.log instead of error.log with my changes. If the suggested change ran on Windows and it couldn't find an error.log, we'd just get more errors about not being able to find crash.log, which to me would be confusing as I'd never heard of crash.log before hitting this issue on Linux.

Reply:
Ideally, the logs would be named the same, but I'm not sure or not if that's something we control.

Reply:
I could add a crash_log() helper to each of the platform resource locator modules (e.g. o3de\Tools\LyTestTools\ly_test_tools\_internal\managers\platforms\windows.py) and then utilize a simple workspace.paths.crash_log() with no special platform callouts.

Reply:
The reason can be added as a comment. I agree that grabbing either file (when present) makes sense.